integration/spark/src/main/scala/org/apache/spark/sql/execution/command/partition/CarbonAlterTableAddHivePartitionCommand.scala

```diff
@@ -137,6 +153,19 @@ case class CarbonAlterTableAddHivePartitionCommand(
         CarbonUtil.checkAndCreateFolderWithPermission(segmentsLoc)
         val segmentPath = segmentsLoc + CarbonCommonConstants.FILE_SEPARATOR + segmentFileName
         SegmentFileStore.writeSegmentFile(segmentFile, segmentPath)
+        CarbonLoaderUtil
+          .recordNewLoadMetadata(newMetaEntry, loadModel, false, false)
+        operationContext.setProperty(table.getTableUniqueName + "_Segment", loadModel.getSegmentId)
```

Comment:
We no need to fire events for SI, since we are not going to load data to SI

Reply:
ok, removed.